src/wp-admin/about.php

```diff
@@ -311,7 +311,11 @@
 					printf(
 						/* translators: %s: WordPress 6.0 Release Notes link. */
 						__( '<a href="%s">Read the WordPress 6.0 Release Notes</a> for more information on the included enhancements and issues fixed, installation information, developer notes and resources, release contributors, and the list of file changes in this release.' ),
-						'#'
+						sprintf(
+							/* translators: %s: WordPress version. */
+							esc_url( __( 'https://wordpress.org/support/wordpress-version/version-%s/' ) ),
+							sanitize_title( '6.0' )
```

Comment:
For clarity and posterity, can you explain why `sanitize_title()` is used on '6.0'? Is this so that the version is filterable using the `sanitize_title` hook?

Reply:
This appears to be a remnant of `sanitize_title( $wp_version )` from an earlier commit: [[44451]](https://core.trac.wordpress.org/changeset/44451). I think it's safe to replace it with just `'6-0'`, the `sanitize_title` hook is probably not super relevant here.

Reply:
Yeah, using `sanitize_title()` was simply a copy, paste and replace. I'll update this.

Reply:
Fixed in e576dff66ce554b9922032a303f86152c577ebb1